src/shims/unix/android/thread.rs

```diff
@@ -0,0 +1,53 @@
+use rustc_span::Symbol;
+use rustc_target::spec::abi::Abi;
+
+use crate::helpers::check_min_arg_count;
+use crate::shims::unix::thread::EvalContextExt as _;
+use crate::*;
+
+const TASK_COMM_LEN: usize = 16;
+
+pub fn prctl<'tcx>(
+    this: &mut MiriInterpCx<'tcx>,
+    link_name: Symbol,
+    abi: Abi,
+    args: &[OpTy<'tcx>],
+    dest: &MPlaceTy<'tcx>,
+) -> InterpResult<'tcx> {
+    // We do not use `check_shim` here because `prctl` is variadic. The argument
+    // count is checked bellow.
+    this.check_abi_and_shim_symbol_clash(abi, Abi::C { unwind: false }, link_name)?;
+
+    let pr_set_name = this.eval_libc("PR_SET_NAME");
+    let pr_get_name = this.eval_libc("PR_GET_NAME");
```

Comment:
Please just hard-code the values for now, with a FIXME saying we can use the values from libc once https://github.com/rust-lang/libc/pull/3941 gets backported to the 0.2 branch.